src/GridView/index.js

```diff
@@ -33,15 +45,26 @@ class GridView extends Container {
         this._filterAnimationFrame = null;
         this._filterCanceled = false;
 
+        // Default options for GridView layout
+        this._multiSelect = args.multiSelect;
```

Comment:
Wasn't gridview multiselect by default before?  Shouldn't these default to true if args doesn't specify them? (Also, that's what I understood from the doc above).

So you'd want something like:
```javascript
this._multiSelect = args.hasOwnProperty('multiSelect') ? args.multiSelect : true;
```